end_to_end/tpu/eval_assert.py

```diff
@@ -0,0 +1,139 @@
+"""
```

Comment:
Let us stage this as a `move` that you do simultaneously in the two repos. The problem with this is that your'e creating duplication in this repo so while the repo is in the duplicated state it isn't clear how other MaxText devs should behave.

Reply:
The ultimate goal is to remove the scripts under `end-to-end` so there will only be two folders `tpu` and `gpu`. I am keeping the original scripts because XLML would break, I have another PR(https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/243/files) in XLML to read the new folders. After that PR is merged, I can delete the scripts under `end-to-end`.

Reply:
Right. Nina and I discussed. Here is the migration plan:

We'd move all the test_*.sh files into maxtext/end_to_end/tpu/. To make sure we don't break existing XLML tests, we shall:

1. Have a PR to add test_*.sh files to maxtext/end_to_end/tpu/
2. Update XLML tests to read from the new path maxtext/end_to_end/tpu/
3. Have another PR to remove test_*.sh files from maxtext/end_to_end/